File: Leetcode/isAnagram.swift
func isAnagram(s: String, _ t: String) -> Bool {
        if s.characters.count != t.characters.count {
            return false
        }
        var dict: [Character : Int] = [:]
        for var ch in s.characters {
            if let value = dict[ch] {
                dict[ch] = value + 1
            } else {
                dict[ch] = 1
            }
        }
        for var ch in t.characters {
            if let value = dict[ch] {
                if value == 0 {
                    return false
                }
                dict[ch] = value - 1
            } else {
                return false
            }
        }
        return true
    }

File: Leetcode/Python/isAnagram.py
def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if s is None or t is None:
            return False
        elif len(s) != len(t):
            return False
        else:
            ls = list(s)
            lt = list(t)
            ls.sort()
            lt.sort()
            return ls == lt
